### ISSUE TITLE
Focus Outline is Black in Some Browsers

### ISSUE THREAD

**Issue by enowak1031** [trimble-oss/website-modus-bootstrap.trimble.com#95]

**Comment by coliff** [comment]
The fix for this is adding this CSS which I'll make in a PR after dark-mode is finished and merged:

```
a:focus-visible {
  outline: 1px solid #0063A3;
}
```
